packages/trip-form/i18n/en-US.yml

```diff
@@ -34,7 +34,6 @@ otpUi:
     optimizeBikeSpeed: Speed
     optimizeFor: Optimize for
     optimizeQuick: Speed
```

Comment:
These two strings should be removed too, they are for the "Optimize Walk" selector.

Reply:
Correction, only `optimizeQuick` should be removed here.